SecondaryInstallScript.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+if [ $USER -ne "root"]
+ 	then
+ 	echo "Permission Denied"
+ 	exit 1
+fi
+
+ping -q -c1 google.com > /dev/null
+ 
+if [ $? -ne 0 ]
+then
+	echo "No Internet Connection...Aborting"
+	exit 1
+fi
+
+#install Epsilon-Onboard-Media-Control
+(cd /usr/local && git clone https://github.com/UCSolarCarTeam/BackupCamera.git)
+cd /usr/local/BackupCamera/Installer
+./MainInstaller.sh
+./AutoBootSetup.sh
+tvservice -d edid 
+edidparser edid
```

Comment:
What is `tvservice` and `edidparser`? Could probably use a comment on what these do.